[2025-03-25T23:30:03.864+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T23:30:03.895+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelada.get_all_data scheduled__2025-03-25T23:00:00+00:00 [queued]>
[2025-03-25T23:30:03.921+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelada.get_all_data scheduled__2025-03-25T23:00:00+00:00 [queued]>
[2025-03-25T23:30:03.922+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2025-03-25T23:30:03.962+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_all_data> on 2025-03-25 23:00:00+00:00
[2025-03-25T23:30:03.996+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=351) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T23:30:04.006+0000] {standard_task_runner.py:72} INFO - Started process 357 to run task
[2025-03-25T23:30:03.998+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelada', 'get_all_data', 'scheduled__2025-03-25T23:00:00+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmp7deo9yqo']
[2025-03-25T23:30:04.007+0000] {standard_task_runner.py:105} INFO - Job 143: Subtask get_all_data
[2025-03-25T23:30:04.183+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelada.get_all_data scheduled__2025-03-25T23:00:00+00:00 [running]> on host d76bf457b908
[2025-03-25T23:30:04.852+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelada' AIRFLOW_CTX_TASK_ID='get_all_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-25T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-25T23:00:00+00:00'
[2025-03-25T23:30:04.859+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-25T23:30:04.883+0000] {inaturalist_dag.py:150} INFO - Getting data from keyword: atropelada
[2025-03-25T23:30:05.188+0000] {inaturalist_dag.py:41} INFO - Maximum id 266353661 for the request
[2025-03-25T23:30:08.360+0000] {inaturalist_dag.py:58} INFO - Got 0 results
[2025-03-25T23:30:09.017+0000] {inaturalist_dag.py:158} INFO - No more observations found, stopping.
[2025-03-25T23:30:10.213+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-25T23:30:10.441+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-03-25T23:30:48.080+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-25T23:30:48.171+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelada, task_id=get_all_data, run_id=scheduled__2025-03-25T23:00:00+00:00, execution_date=20250325T230000, start_date=20250325T233003, end_date=20250325T233048
[2025-03-25T23:30:48.165+0000] {job.py:229} INFO - Heartbeat recovered after 37.74 seconds
[2025-03-25T23:30:49.169+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T23:30:49.783+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-25T23:30:49.793+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
